app/src/main/java/org/systers/mentorship/utils/PreferenceManager.kt

```diff
@@ -31,4 +31,11 @@ class PreferenceManager {
 
     val authToken: String
         get() = sharedPreferences.getString(AUTH_TOKEN, "")
+
+    /**
+     * Clears SharedPreferences file
```

Comment:
give a detailed description like 
`Clears all the data that has been saved in the preferences file.`